032-projects/proposal.md

```diff
@@ -255,6 +255,59 @@ Instead, these should be treated as two separate inputs. Your product code is an
 TODO: finish this thought - wrap it up by suggesting that 
 
 
+# Open Questions
+
+## Can Projects replace Teams?
```

Comment:
Please don't? 😄 

We use the Teams concept to map onto our product teams. The term "project" is considered a legacy, non-agile term. As such, the first question replacing Teams by Projects would raise in our organization is: "It no longer supports agile organizations?" (I know, I know... still...) We barely convinced people that agile is good and barely got people used to product teams, etc.

I was kinda hoping Projects would become an organizational tool under Teams for housing related Pipelines. I.e. you'd have a Team like "Tasty Treats app" which could have Projects like "Strawberry Icecream", "Bubble Tea" and "Apple Pie" with Pipelines underneath those related to the various tasks and builds that need doing there.

But feel free to ignore my monologue. Just dropping my 2 cents. 😄